<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Clasament</class>
 <widget class="QDialog" name="Clasament">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>366</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(170, 170, 127);
</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>107</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Categoria de greutate</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>68</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Categoria gen</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>96</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Categoria de varsta</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_back">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>51</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(247, 247, 182);</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>230</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(245, 245, 184);</string>
   </property>
   <property name="text">
    <string>Afisati clasamentul</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>226</x>
     <y>82</y>
     <width>121</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="b_viewclasament">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>300</y>
     <width>201</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(247, 247, 182);</string>
   </property>
   <property name="text">
    <string>Afisati clasament organizatii</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(247, 247, 182);</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(247, 247, 182);</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(247, 247, 182);</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
